app/src/bin/crate.bat

```diff
@@ -65,7 +65,13 @@ REM JAVA_OPTS=%JAVA_OPTS% -XX:HeapDumpPath=$CRATE_HOME/logs/heapdump.hprof
 REM Ensure UTF-8 encoding by default (e.g. filenames)
 set JAVA_OPTS=%JAVA_OPTS% -Dfile.encoding=UTF-8
 
-set CRATE_CLASSPATH=%CRATE_CLASSPATH%;%CRATE_HOME%/lib/crate-*.jar;%CRATE_HOME%/lib/*;%CRATE_HOME%/lib/sigar/*
+if "%CRATE_CLASSPATH%" == "" (
```

Comment:
also change the comments in bin/crate https://github.com/crate/crate/blob/s/jar-hell/app/src/main/dist/bin/crate#L11